apps/website/src/theme/Navbar/ColorModeToggle/index.tsx

```diff
@@ -1,24 +1,23 @@
-import React from 'react';
-import { useColorMode } from '@docusaurus/theme-common';
+import * as React from 'react';
+import { useColorMode, useThemeConfig } from '@docusaurus/theme-common';
 
-import useIsBrowser from '@docusaurus/useIsBrowser';
-import { GithubIcon, MoonIcon, SunIcon } from '../../components/icons';
+import { GithubIcon, MoonIcon, SunIcon } from '../../../components/icons';
 import styles from './styles.module.css';
 
 function ThemeToggle() {
-  const disabled = useIsBrowser() === false;
-  const { isDarkTheme, setLightTheme, setDarkTheme } = useColorMode();
+  const disabled = useThemeConfig().colorMode.disableSwitch;
+  const { colorMode, setColorMode } = useColorMode();
```

Comment:
- The component was renamed
- `isDarkTheme` => `colorMode`, `setLightTheme` => `setColorMode`